src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -506,5 +553,7 @@ export default connect(
     addSelectedAnswers: addSelectedAnswersToState,
     removeAnswer: removeSelectedAnswer,
     createNewAnswerRow: addNewAnswerRow,
+    deleteConcept: removeDictionaryConcept,
```

Comment:
How are these two new lines related to the ticket?

Reply:
A reference to a concept cannot be edited directly, therefore, it has to first be deleted from the Dictionary, then Recreated. This prop is being called within the method: `updateConceptReference()`